source/OptimaJet.DWKit.StarterApplication/Startup.cs

```diff
@@ -103,11 +103,11 @@ public virtual void ConfigureDatabase(IServiceCollection services, string name =
 
         private string ConfigureConnectionString()
         {
-            var dbHost = GetVarOrThrow("POSTGRES_HOST");
+            var dbHost = GetVarOrDefault("POSTGRES_HOST", "localhost");
             var dbPort = GetVarOrDefault("POSTGRES_PORT", "5432");
-            var dbDatabase = GetVarOrThrow("POSTGRES_DB");
-            var dbUser = GetVarOrThrow("POSTGRES_USER");
-            var dbPassword = GetVarOrThrow("POSTGRES_PASSWORD");
+            var dbDatabase = GetVarOrDefault("POSTGRES_DB", "development");
+            var dbUser = GetVarOrDefault("POSTGRES_USER", "db-user");
+            var dbPassword = GetVarOrDefault("POSTGRES_PASSWORD", "1234");
```

Comment:
These are needed so that the `dotnet ef migrations script --idempotent` command can run during build-time.

Reply:
I mean, the .env file is typically source'd into the current shell environment before running. :-\

Reply:
During building of the Docker image, there is no .env file, right?

Reply:
you can load .env with bash. 

```
set -a
source .env
dotnet ...
```